Redesign and clean project homepage (#113)
pymc-devs/pymc.io: welcome.md

  </div>
  <div class="homepage-button-container-row">
      <a href="https://www.pymc.io/project/docs/en/stable/api.html" class="homepage-button-link">See API Reference →</a>
  </div>
</div>
</div>
</div> 

<div style="display: flex; max-width: 55rem; margin: auto;">
<h3 style="font-size: 36px; font-weight: normal; text-align: center;">
PyMC is a probabilistic programming library for Python that allows users to build Bayesian models with a simple API. It features multiple inference algorithms, forward sampling, and model updates or interventions.
</h3>
</div>

## Built for insight

:::::{grid} 1 2 3 3

::::{grid} 1
:::{grid-item}
:class: key-features-icon

{material-twotone}`data_exploration`
:::
:::{grid-item}
:class: key-features-name

Modern
:::
:::{grid-item}
:class: key-features-body

Includes state-of-the-art inference algorithms, including MCMC (NUTS) and variational inference (ADVI).
:::
::::

::::{grid} 1
:::{grid-item}
:class: key-features-icon

{material-twotone}`how_to_reg`
:::
:::{grid-item}
:class: key-features-name

User-friendly
:::
:::{grid-item}
:class: key-features-body

> Write your models using friendly Python syntax. [Learn Bayesian modeling](https://www.pymc.io/projects/docs/en/latest/learn.html#) from the many [example notebooks](https://www.pymc.io/projects/examples/en/latest/gallery.html).

Reviewer: Following @twiecki   suggestion, we could have something like 

```suggestion
Build models from intuitive primitives called random variables. For example, `x = pm.Normal("x", mu=0, sigma=1) defines an unknown quantity x with a Normal distribution.
```

And we can move the text [Learn Bayesian modeling](https://www.pymc.io/projects/docs/en/latest/learn.html#) from the many [example notebooks](https://www.pymc.io/projects/examples/en/latest/gallery.html). to some other place, maybe add a new "card" "easily to learn" or "well-documented" saying that there are a lot of resources to learn, videos, books, tutorials, etc...

Reviewer 2: I like that a lot, especially the first sentence.

> Write your models using friendly Python syntax. [Learn Bayesian modeling](https://www.pymc.io/projects/docs/en/latest/learn.html#) from the many [example notebooks](https://www.pymc.io/projects/examples/en/latest/gallery.html).

Reviewer: Following @twiecki   suggestion, we could have something like 

```suggestion
Build models from intuitive primitives called random variables. For example, `x = pm.Normal("x", mu=0, sigma=1) defines an unknown quantity x with a Normal distribution.
```

And we can move the text [Learn Bayesian modeling](https://www.pymc.io/projects/docs/en/latest/learn.html#) from the many [example notebooks](https://www.pymc.io/projects/examples/en/latest/gallery.html). to some other place, maybe add a new "card" "easily to learn" or "well-documented" saying that there are a lot of resources to learn, videos, books, tutorials, etc...

Author: I adapted the elements in the "key features grid" from the old bullet points, but we can change/modify those as much as we do their descriptions, also add more and have a 4x2 grid, or 3x3

:::
::::

::::{grid} 1
:::{grid-item}
:class: key-features-icon

{material-twotone}`speed`
:::
:::{grid-item}
:class: key-features-name

Fast
:::
:::{grid-item}
:class: key-features-body

>  Uses {doc}`PyTensor <pytensor:index>` as its computational backend to compile through C, Numba or JAX, [run your models on the GPU](https://www.pymc-labs.io/blog-posts/pymc-stan-benchmark/), and benefit from complex graph-optimizations.

Reviewer: ```suggestion
PyMC runs fast even for complex models. Thanks to a high-performance computational graph engine, {doc}PyTensor <pytensor:index>, your models are compiled through C, Numba, or JAX to run on CPU or GPU and benefit from automatic graph optimizations—without you even noticing.
```
Or something like this to have a balance between technical language and punchier language
